lib/wor/paginate/adapters/kaminari.rb

```diff
@@ -13,6 +13,10 @@ def paginated_content
           @paginated_content ||= @content.page(@page).per(@limit)
         end
 
+        def previous_page
```

Comment:
You define in this way, and use `delegate` in the next file `lib/wor/paginate/adapters/will_paginate.rb`. Unify the methods, I recommend use delegate

Reply:
I delegated it in the `lib/wor/paginate/adapters/will_paginate.rb` because Will Paginate has his own `previous_page`method tied to an ActiveRecord Relation, is the same thing that happens with the `next_page` method, which was delegated already. In the Kaminari adapter I can't do de same, because the method that has for an ActiveRecord Relation is called `prev_page` which is different than the main call of the method `previous_page` so that's why I had to do sort of an "override" in here. How you suggest I can unify methods, given that both previous_page methods in both adapters are differently named?

Reply:
Actually I made a main method for calculate the previous_page in the Base adapter which is the father of all adapters, (same as next_page method). But I noticed that in the case of Will Paginate, it is preferred to use the Will Paginate next_page method of an ActiveRecord Relation instead of using the one in the Base adapter. So I did the same for previous_page method: a main Base method, but Will Paginate/Kaminari previous_page method is preferred over than the main method in the Base. (I could leave just the previous_page in the Base that do all the work, and delete those "override" methods, but would be different than the implementation of `next_page`... so I don't know which is better

Reply:
Ohhhh I don't saw you call prev_page... My bad :man_facepalming: